services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/email/EmailSenderService.kt

```diff
@@ -14,11 +14,14 @@ private val logger = KotlinLogging.logger {}
 @Service
 class EmailSenderService(
     private val mailSender: JavaMailSender,
-    @Value("\${haitaton.email.enabled}") private val enabled: Boolean,
+    @Value("\${haitaton.email.filter.use}") private val sendOnlyForAllowed: Boolean,
+    @Value("\${haitaton.email.filter.allow-list}") private val allowListRaw: String,
```

Comment:
You can use SpEL to split the property to a list directly: https://www.baeldung.com/spring-inject-arrays-lists#using-custom-delimiters

Reply:
Nice, fixed.